src/dapps/selectors.ts

```diff
@@ -51,8 +54,11 @@ export const dappCategoriesByIdSelector = createSelector(
       }
     })
     dapps.forEach((dapp) => {
-      mappedCategories[dapp.categoryId]?.dapps.push(dapp)
+      if (!favoriteDappIds.includes(dapp.id)) {
+        mappedCategories[dapp.categoryId]?.dapps.push(dapp)
+      }
```

Comment:
A comment here would be nice, to explain why we're doing this.

Reply:
hmmm i added a comment as you suggested but not sure why it's not showing up in this diff, it is showing up on the file if you browse the branch though https://github.com/valora-inc/wallet/blob/9332286834a5e1c023b22e1eb6c564a0a110069e/src/dapps/selectors.ts#L57 🙈 take my word for it